tests/common/db/malware.py

```diff
@@ -51,6 +51,12 @@ class Meta:
 
     check = factory.SubFactory(MalwareCheckFactory)
     release_file = factory.SubFactory(FileFactory)
+    release = None
```

Comment:
Would it be possible to add `SubFactory` selectors for these?

Reply:
It would, but I'm not sure what effect it would have. A verdict should only be associated with one of these fields at most.

Reply:
Gotcha, makes sense!